docs/source/worker.rst

```diff
@@ -233,6 +233,12 @@ YARN, Mesos, SGE, etc..).  After termination the nanny will restart the worker
 in a fresh state.
 
 
+Nanny
+~~~~~
+
```

Comment:
```suggestion
Dask workers are often launched, monitored, and managed by a small Nanny process.
```

Then, I would add more context into the Nanny docstring itself.  In particular, we should probably add a See Also section that links to the worker, and we should mention that the parameters to Nanny are mostly the same as the parameters that are in the Worker.